deploy/cloudformation/static-host-pipeline.yml

```diff
@@ -376,7 +418,12 @@ Resources:
                 - aws s3 rm s3://$DEST_BUCKET --recursive
             build:
               commands:
-                - aws s3 cp --recursive . s3://$DEST_BUCKET/
+                - SOURCE_SHA=$(cat sha.txt)
+                - aws s3 cp --recursive . s3://$DEST_BUCKET/ --exclude "sha.txt"
+                - DATE=$(date)
+            post_build:
+              commands:
+                - curl -i -X POST -H "Authorization:token ${OAUTH_TOKEN}" "https://api.github.com/repos/ndlib/mellon-image-viewer/statuses/${SOURCE_SHA}" -d "{\"state\":\"success\",\"description\":\"Deployed to production on ${DATE}\",\"context\":\"${CODEBUILD_INITIATOR}\",\"target_url\":\"https://image-viewer.library.nd.edu\"}"
```

Comment:
Now that I'm actually able to look from my computer, I think we need to parameterize the repo name and URL_TO_PRODUCTION since this pipeline is supporting multiple components of the mellon project.